src/core/constants.js

```diff
@@ -15,14 +15,28 @@ export const VERSION = 'VERSION_WILL_BE_REPLACED_BY_BUILD';
 
 // GRAPHICS RENDERER
 /**
- * The default, two-dimensional renderer.
- * @typedef {unique symbol} P2D
+ * The default, two-dimensional renderer in p5.js.
+ *
+ * Use this when calling {@link p5.createCanvas|createCanvas} (for example,
```

Comment:
I'm not sure that we support `@link` yet, this is the result of `npm run docs`:
<img width="1248" alt="image" src="https://github.com/user-attachments/assets/7bcf445b-3ad2-49ae-a776-7295fb8d2381" />

It looks like we don't use this syntax elsewhere in the codebase currently, could we keep using the old `<a href="#/p5/createCanvas">` for consistency?